include/ginkgo/core/solver/gmres.hpp

```diff
@@ -31,6 +31,29 @@ namespace solver {
 
 constexpr size_type gmres_default_krylov_dim = 100u;
 
+namespace gmres {
+/**
+ * Set the orthogonalization method for the Krylov subspace.
+ */
+enum class orthog_method {
```

Comment:
Ha, I went with `orthog` because `ortho` just makes me think of teeth, rather than orthogonalization :) 
Should we have a poll? Maybe someone has another better option than either of these?

Reply:
:) I dont have a strong preference. Both are okay with me.